src/fidesctl/cli/__init__.py

```diff
@@ -89,7 +89,8 @@ def cli(ctx: click.Context, config_path: str, local: bool) -> None:
     # Check the server health and version if an API command is invoked
 
     if ctx.invoked_subcommand in SERVER_CHECK_COMMAND_NAMES:
-        check_server(VERSION, config.cli.server_url)
+        # suppress output unless there's an error
```

Comment:
This comment feels like bloat. I think the inclusion of `quiet=True` in the arg list is perfectly clear on its own, no?